packages/calcite-components/src/components/accordion-item/resources.ts

```diff
@@ -17,6 +18,8 @@ export const CSS = {
   iconStart: "icon--start",
   iconEnd: "icon--end",
   headerContainer: "header-container",
+  iconPosition: (position: string): string => `icon-position--${position}`,
```

Comment:
I think I mentioned this before, but I'm digging this approach. 

Sidebar: I thought we had an entry for this pattern under conventions, but it doesn't seem like it. I'll create a doc-update issue.

Reply:
https://github.com/Esri/calcite-design-system/issues/9117